### PR TITLE
Ajout de la gestion des groupes pour un user PostgreSQL

### PR DESCRIPTION
**Pull request by Vidock-pro**

### CODE REVIEW

File: tasks/users_membership.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    group: "{{item.groups}}"
+    db: "{{item.db | default(omit)}}"
+    port: "{{postgresql_port}}"
+    target_roles: "{{item.name}}"
+    state: present
+    login_host: "{{item.host | default(omit)}}"
+    login_user: "{{postgresql_admin_user}}"
+  become: yes
+  become_user: "{{postgresql_admin_user}}"
+  with_items: "{{postgresql_user_membership}}"
```
Comment (lrk) [inline tasks/users_membership.yml:14]: Please add default empty array `postgresql_user_membership` in `defaults/main.yml` with comment describing how to use that variable

File: tasks/main.yml [hunk 2]
```diff
@@ -48,6 +48,9 @@
 - import_tasks: users_privileges.yml
   tags: [postgresql, postgresql-users]
 
+- import_tasks: users_membership.yml
```
[the remark this answers is not in the document]
Comment (Vidock-pro) [reply tasks/main.yml:51]: resolve in the commit 315e5aba4994026dfbc0b6761e441d3a8d804291

File: tasks/users_membership.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# file: postgresql/tasks/users_membership.yml
+
+- name: PostgreSQL | Update the user membership
+  postgresql_membership:
+    group: "{{item.groups}}"
+    db: "{{item.db | default(omit)}}"
+    port: "{{postgresql_port}}"
+    target_roles: "{{item.name}}"
+    state: present
```
[the remark this answers is not in the document]
Comment (Vidock-pro) [reply tasks/users_membership.yml]: resolve in the commit 315e5aba4994026dfbc0b6761e441d3a8d804291

File: tasks/users_membership.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    login_host: "{{item.host | default(omit)}}"
+    login_user: "{{postgresql_admin_user}}"
```
[the remark this answers is not in the document]
Comment (Vidock-pro) [reply tasks/users_membership.yml:12]: resolve in the commit 315e5aba4994026dfbc0b6761e441d3a8d804291

File: tasks/users_membership.yml (hunk 1 above)
Comment (Vidock-pro) [reply tasks/users_membership.yml]: i've changer to use the existant var postgresql_user_privileges

**Pull request merged**